src/Adapter/Translations/TranslationRouteFinder.php

```diff
@@ -72,19 +75,27 @@ class TranslationRouteFinder
      */
     private $moduleRepository;
 
+    /**
+     * @var TranslatorInterface
+     */
+    private $translator;
+
     /**
      * @param TranslationService $translationService
      * @param Link $link
      * @param ModuleRepositoryInterface $moduleRepository
+     * @param TranslatorInterface $translator
      */
     public function __construct(
         TranslationService $translationService,
         Link $link,
-        ModuleRepositoryInterface $moduleRepository
+        ModuleRepositoryInterface $moduleRepository,
+        TranslatorInterface $translator
```

Comment:
I'm not sure about injecting the translator here. In addition, it would be considered a breaking change.

Reply:
I completely agree ! 
Why make it complicated when it can be simple ? 😀